### PR TITLE
feat: Kubernetes seminar slides

### PR DESCRIPTION
**Pull request by anitvam**

### CODE REVIEW

File: content/extra/kubernetes/_index.md [hunk 1]
```diff
@@ -0,0 +1,1261 @@
...
+
+## Access Control
+
+Enforced through the `Service Account` resource object
+  - represents a distinct identity in the cluster
+  - each one of them is *bound* to a specific `Namespace`
+  - permit to access the API server
+  - grants permissions following the *RBAC* mechanism
+  - can be used to configure the authentication within `kubectl`
+
```
Comment (DanySK) [inline content/extra/kubernetes/_index.md:734]:
```suggestion
Enforced through the `Service Account` resource object
- represents a distinct identity in the cluster
- each one of them is *bound* to a specific `Namespace`
- grants access to the API server
- grants permissions through the *RBAC* mechanism
- can be used to configure the authentication within `kubectl`

```

File: content/extra/kubernetes/_index.md [hunk 2]
```diff
@@ -0,0 +1,1261 @@
...
+## Kubectl config: contexts
+
+- Contexts declare the user to use to connect to a specific cluster.
+- You can define multiple contexts to connect to clusters.
+- You can connect to a single context at a time, 
+  - and switch between them using the `kubectl config use-context` command.
+<div class="custom">
+
+```bash
```
Comment (DanySK) [inline content/extra/kubernetes/_index.md:1032]:
```suggestion
```yaml
```

File: content/extra/kubernetes/_index.md [hunk 3]
```diff
@@ -0,0 +1,1261 @@
...
+## Let's test the Kubernetes autoscaling
+
+Now we can deploy our first Application
+  - it's a simple php web server that answer "OK!" to every request
+
+
+<br>
+<div class="center">
+<b>What kubernetes objects should we use to deploy it?</b>
+</div>
```
Comment (DanySK) [inline content/extra/kubernetes/_index.md:1099]:
```suggestion
#### **What kubernetes objects should we use to deploy it?**
```

**Review by DanySK — COMMENTED** [review]
I would limit the usage of html as much as possible, but whatever. Decide whether or not to accept the suggestions, and let me know when ready to merge.
Well written, well done.

File: content/extra/kubernetes/_index.md [hunk 4]
```diff
@@ -0,0 +1,1261 @@
...
+.reveal pre code {
+  font-size: 1.2em;
+}
+
+.center{
+	text-align: center;
+}
+
+
+</style>
```
[the remark this answers is not in the document]
Comment (anitvam) [reply content/extra/kubernetes/_index.md:42]: I thought about that but I was worried to break your slides, so I adapted mine.

**Pull request merged**